src/luarocks/admin/cmd/add.lua

```diff
@@ -11,20 +11,21 @@ local fs = require("luarocks.fs")
 local cache = require("luarocks.admin.cache")
 local index = require("luarocks.admin.index")
 
-add.help_summary = "Add a rock or rockspec to a rocks server."
-add.help_arguments = "[--server=<server>] [--no-refresh] {<rockspec>|<rock>...}"
-add.help = [[
-Arguments are local files, which may be rockspecs or rocks.
-The flag --server indicates which server to use.
-If not given, the default server set in the upload_server variable
-from the configuration file is used instead.
-
---no-refresh  The local cache should not be refreshed
-              prior to generation of the updated manifest.
---index       Produce an index.html file for the manifest.
-              This flag is automatically set if an index.html
-              file already exists.
-]]
+function add.add_to_parser(parser)
+   local cmd = parser:command("add", "Add a rock or rockspec to a rocks server.",
+      util.see_also())
```

Comment:
this line wrapping looks weird to me. The luarocks codebase doesn't mind a few longer lines.